runner.ts

```diff
@@ -49,6 +49,39 @@ export async function run(
   source: string,
   config: Config
 ): Promise<[Value, compiler.GlobalEnv, GlobalTypeEnv, string]> {
+  // One approach to create built-in Range class and range function, by Comprehension team
+  if (!config.typeEnv.classes.has("Range")) {
```

Comment:
Is there a way to do this without concatenating code?

Reply:
@jpolitz Currently, we followed this (rather unclean) approach, since our current implementation is kind of a temporary "hack" and only supports `range(...)` calls as iterators. 

But our ideal end goal is to replace all this by supporting lists natively, without the need for any sort of built-in functions/classes, like `range(...)` and `Range`.

So we were thinking that, at that point, we would just remove this part here.